packages/components/managed-data-table/README.md

```diff
@@ -0,0 +1,86 @@
+# Table
```

Comment:
In this Readme I would rather just document the features added by `ManagedTable` in relation to the simpler `DataTable`, without duplicating the documentation of the DataTable but referring to its documentation instead (as well as the other components that are part of ManagedTable, such as the ColumnManager).
This would avoid having to duplicate every change to the docs of DataTable to this file, which can easily go out of hand.